src/components/AirdropWalletBalanceCard.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+
+import { CONST, AirdropWalletDetails, AirdropGoal } from 'app/consts';
+import { isAfterAirdrop } from 'app/helpers/airdrop';
+import { typography, palette } from 'app/styles';
+
+import { AirdropWalletBalance } from './AirdropWalletBalance';
+
+const i18n = require('../../loc');
+
+interface Props {
+  walletDetails: AirdropWalletDetails;
+}
+
+export const AirdropWalletBalanceCard: FC<Props> = ({ walletDetails }) => {
```

Comment:
Did you check the edge case when the threshold is equal to the balance, before and after the airdrop if it is displayed correctly?

Reply:
yup. When threshold is equal, it takes renders higher threshold as a goal, so all fine. 
Also put `true` in place of `isAfterAirdrop` and all seemed fine as well. Pushed adjusted 'after airdrop reached goal' inclusion just now.